modules/commands/ns_drop.cpp

```diff
@@ -45,10 +45,14 @@ class CommandNSDrop : public Command
 			source.Reply(_("You may not drop other Services Operators' nicknames."));
 		else
 		{
+			User *u = source.GetUser();
```

Comment:
`source` could be a services operator force dropping a nick. For future reference, `source.GetUser()` could also return NULL when a command is called programmatically (i.e., web panel (drop could be implemented someday) or a custom module, etc.)